### ISSUE TITLE
f8-wildfly-swarm-health-check no longer works

### ISSUE THREAD

**Issue by davsclaus** [fabric8io/fabric8-maven-plugin#896]
It used to work with 3.2.9 version. But latest 3.3.0 and also 3.2.31 does not work.

This project
https://github.com/davsclaus/fabric8-hello/tree/master/helloswarm

Works with 3.2.9
```
[INFO] --- fabric8-maven-plugin:3.2.9:resource (default-cli) @ helloswarm-kubernetes ---
Downloading: https://repo.maven.apache.org/maven2/org/apache/camel/camel-parent/2.18.1/camel-parent-2.18.1.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/camel/camel-parent/2.18.1/camel-parent-2.18.1.pom (187 KB at 136.4 KB/sec)
Downloading: https://repo.maven.apache.org/maven2/org/apache/camel/camel/2.18.1/camel-2.18.1.pom
Downloaded: https://repo.maven.apache.org/maven2/org/apache/camel/camel/2.18.1/camel-2.18.1.pom (28 KB at 436.4 KB/sec)
[INFO] F8: Running in Kubernetes mode
[INFO] F8: Running generator wildfly-swarm
[INFO] F8: Using resource templates from /Users/davsclaus/workspace/camelinaction2/chapter18/hello-kubernetes/hello-swarm2/src/main/fabric8
2017-03-24 12:59:37 INFO  Version:30 - HV000001: Hibernate Validator 5.2.4.Final
[INFO] F8: fmp-controller: Adding a default Deployment
[INFO] F8: wildfly-swarm-health-check: Adding readiness probe on port 8080, path='/health', scheme='HTTP', with initial delay 10 seconds
[INFO] F8: wildfly-swarm-health-check: Adding liveness probe on port 8080, path='/health', scheme='HTTP', with initial delay 180 seconds
```

But fails with latest where the readiness and liveness probes are not added.

**Comment by heiko-braun** [comment]
[16:36:46] ro14nd:	yes, when the enricher is enabled in a profile. the default profile looks like 
[16:36:47] ro14nd:	https://github.com/fabric8io/fabric8-maven-plugin/blob/master/plugin/src/main/resources/META-INF/fabric8/profiles-default.yml
[16:37:06] ro14nd:	just see that the wildlfy-swarm check is not enabled by default.
[16:37:52] ro14nd:	hmm.
[16:41:09] ro14nd:	you can enable it by hand with a <enricher><includes><include>wildfly-swarm-health-check</....
[16:41:59] hbraun:	ro14nd tnx. the automatic discovery would be nice to have
[16:42:15] ro14nd:	yes, of course. guess it was removed by accident

**Comment by Ladicek** [comment]
Is this possibly the same as https://github.com/fabric8io/fabric8-maven-plugin/issues/924, which was fixed for Swarm by https://github.com/fabric8io/fabric8-maven-plugin/pull/930?

**Comment by rhuss** [comment]
yep, thats it at fixed with #930
